akka-docs/rst/scala/code/docs/event/LoggingDocSpec.scala

```diff
@@ -134,67 +161,44 @@ class LoggingDocSpec extends AkkaSpec {
   }
 
   "allow registration to dead letters" in {
-    new AnyRef {
```

Comment:
Removed extra anonymous class. I think that it is needed not to publish LoggingDocSpec instance by using **this** keyword in the `Props(classOf[Listener], this)`